src/org-manager/containers/organisation-details/organisation-details.component.ts

```diff
@@ -0,0 +1,34 @@
+import {Component, OnInit, Input} from '@angular/core';
+import * as fromStore from '../../store';
+import * as fromRoot from '../../../app/store';
+import { Store, select } from '@ngrx/store';
+import { OrganisationVM} from 'src/org-manager/models/organisation';
+import { Observable } from 'rxjs';
+import {map, takeWhile} from 'rxjs/operators';
+
+/**
+ * Bootstraps Organisation Details
+ */
+@Component({
+  selector: 'app-org-details',
+  templateUrl: './organisation-details.component.html'
+})
+export class OrganisationDetailsComponent implements OnInit {
+
+  orgSubscription$: Observable<OrganisationVM[]>;
+  dxNumber: string;
+  dxExchange: string;
+
+  constructor(public store: Store<fromStore.OrganisationRootState>) {}
+
+  ngOnInit(): void {
+    this.orgSubscription$ = this.store.pipe(select(fromStore.selectedActiveOrganisation));
+    this.orgSubscription$.pipe(takeWhile(org => !org), map(console.log)).subscribe(org => {
```

Comment:
This code will not do what you expect, the console.log (which shouldn't be there) breaks this code.

as you use map, the org gets mapped to the return value of console.log, which is always undefined. This means that the org in the subscribe cb is also always undefined...